src/chapters/11-route-params.md

```diff
@@ -95,7 +110,18 @@ Alright, we have just one more step left here: updating the tests. We can add an
 git add tests/integration/components/rental-test.js
 ```
 
-Awesome! However, there is one more point to mention here: by default, a component test (like the one that we have here) does not set up the router. Usually, this is what you'd want, since for more components, you likely aren't going need the router to test what you want. But in this specific case, we have a `<LinkTo>` in our component that is relying on the router to generate its URLs.
+If we run the tests in the browser, everything should...
+
+```run:screenshot width=1024 height=768 retina=true filename=fail.png alt="The test failed"
+visit http://localhost:4200/tests?nocontainer&deterministic
+wait  #qunit-banner.qunit-fail
+```
+
+...wait a minute, it didn't work!
+
+About time something didn't Just Work™ the first time we tried; this is the *advanced* part of the tutorial after all.
+
+By default, a component test (like the one that we have here) does not set up the router. Usually, this is acceptable, but in this specific case, we have a `<LinkTo>` in our component that is relying on the router to generate its URLs.
```

Comment:
Now that I'm reading this, feels weird to go from "it didn't work" to "by default this is how component tests work". Need a sentence here that bridges the gap between the two.

Reply:
Good catch! We're changing it to this: 

"Component tests (like the one we have here) do not set up the router by default, because it's usually not necessary. In this specific case, however, we have a `<LinkTo>` in our component that is relying on the router to generate its URLs."

Let us know if you have any edits for the new paragraph.